### ISSUE TITLE
Confusion from project name and readme

### ISSUE THREAD

**Issue by larrydewey** [IBM/sev-snp-measure#6]
Because of the way the SEV API specification is written, users of this tool might be confused to learn this tool does not generate the expected launch **_measurement_**, but calculates an expected launch _**digest**_

Launch Measurement:
```
# Legacy SEV (non-SNP)
HMAC(0x04 || API_MAJOR || API_MINOR || BUILD || GCTX.POLICY || GCTX.LD || MNONCE; GCTX.TIK)
```

Launch Digest:
```
# Naples - UEFI Possibly additional measurements.
SHA256SUM(OVFM||Linux Kernel||initrd||Command Line Parameters)

# >= Rome - UEFI, Kernel, Initrd, CLI-params, VMSA, Pages (Milan+)
SHA384SUM(OVFM||Linux Kernel||initrd||Command Line Parameters||VMSA)
```

Perhaps some re-naming would be beneficial?

**Comment by dubek** [comment]
hmmm, this is indeed confusing; thanks @larrydewey for pointing this out.

This tool started with SNP support. In the SNP ABI spec, Section 7.3, Table 21, the attestation report contains a field called `MEASUREMENT` described as `the measurement calculated at launch` (it is 384 bits long). This tool aimed to calculate exactly this field.

Later we added SEV and SEV-ES support, but indeed the tool calculates just the `GCTX.LD` part.

[Small note: the digest for SNP is not simply SHA384 of concatenation of pages; it is a more complex scheme (see the GCTX class or the SNP ABI description of `SNP_LAUNCH_UPDATE`).]

**Comment by larrydewey** [comment]
You are correct. My reason for bringing up the issue is for the SEV and SEV-ES workflows. SEV-SNP changes things significantly.

**Comment by dubek** [comment]
Maybe we should add a clarification in the README about SEV/SEV-ES - that the tool actually calculates only the expected launch digest, and doesn't do all the rest of the `LAUNCH_MEASURE` parts.
